### ISSUE TITLE
Support for Capstone Disassembler

### ISSUE THREAD

**Issue by aoshiken** [tklengyel/drakvuf#483]
@tklengyel,
I think adding support for the [Capstone Disassembler](https://github.com/aquynh/capstone) would be very useful for setting traps inside functions because we're currently limited to symbols addresses or harcoded offsets inside these symbols.

The changes would be minimal, just adding the library search in `configure.ac` and perhaps checking for a minimal version (2.0 if you ask me).

Probably I'm shooting before loading the gun but... thoughts?

**Comment by tklengyel** [comment]
Yes, if there is a particular usecase that requires trapping inside functions or programs without debug data, Capstone would be a good choice to use for dissassembly.
